torch/_dynamo/utils.py

```diff
@@ -965,6 +966,16 @@ def istype(obj, allowed_types):
     )
 
 
+def get_inputs_devices(
+    inputs: collections.abc.Sequence[object],
+) -> list[Optional[torch.device]]:
+    return list(
+        OrderedSet(
```

Comment:
Shouldn't we use all devices in graph, not just input ?

Reply:
Thanks for the comment. Added FX Graph Module in to account.